app-misc/openrazer/openrazer-3.2.0.ebuild

```diff
@@ -61,6 +61,10 @@ MODULE_NAMES="
 
 src_prepare() {
 	default
```

Comment:
Please change the order so that `default` is run after the patching of the `.desktop`.